### PR TITLE
Style updates

### PR DESCRIPTION
**Pull request by kiverso**
Updates styling on browse, dashboard, and search. Creates thumbnails for browse and search. Update redirect path for approvals controller update action.

### CODE REVIEW

**Comment by muydanny** [comment]
we did it

File: db/seeds.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
 mad_money = Podcast.create(name: "Mad Money w/ Jim Cramer", description: mad_money_description , location: 'Dallas', active: false, twitter: '@madmoney', instagram: '@madmoney', facebook: '@madmoney', photo: "https://cdn-images-1.listennotes.com/podcasts/mad-money-w-jim-cramer-cnbc-qzcgTHGP63V-a8p6YXsZcxV.300x300.jpg", user_id: users[3].id, adult_content: true, podcast_uri: 'e3bb1e6c39d245f081a4a4db8ce9b661' )
 
-rnz = Podcast.create(name: "RNZ: Flying Solo", description: rnz_description , location: 'San Fransisco', active: true, twitter: '@RNZ', instagram: '@RNZ', facebook: '@RNZ', photo: "https://cdn-images-1.listennotes.com/podcasts/rnz-flying-solo-radio-new-zealand-jCrRmkPl6Gn.300x300.jpg", user_id: users[1].id, adult_content: false, podcast_uri: '550e80e1c4a64bd290493b7bdbb4458c' )
+rnz = Podcast.create(name: "RNZ: Flying Solo", description: rnz_description , location: 'San Francisco', active: true, twitter: '@RNZ', instagram: '@RNZ', facebook: '@RNZ', photo: "https://cdn-images-1.listennotes.com/podcasts/rnz-flying-solo-radio-new-zealand-jCrRmkPl6Gn.300x300.jpg", user_id: users[1].id, adult_content: false, podcast_uri: '550e80e1c4a64bd290493b7bdbb4458c' )
```
Comment (janegreene) [inline db/seeds.rb:29]: Ah! Thanks for catching this.

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1,27 +1,31 @@
...
+      </div>
+    <% end %>
+  </div>
 </section>
-
-<h6 class="center-search"> Interested in becoming a contributor with Podsmack? Apply to add your show here:  </h6>
-<div class="center-search">
-  <%= button_to 'Upload your own podcast', search_path, method: 'get', class: "btn btn-outline btn-xl js-scroll-trigger" %>
+<div class='container'>
+  <p> Interested in becoming a contributor with Podsmack?</p>
```
Comment (janegreene) [inline app/views/users/show.html.erb:14]: I like this better, nice job.

**Pull request merged**